.maintain/frame-weight-template.hbs

```diff
@@ -0,0 +1,103 @@
+// This file is part of Substrate.
```

Comment:
the license needs to be adjusted

Reply:
@GopherJ I don't think so. We should not change this since this file has been taken from https://github.com/paritytech/substrate/blob/master/.maintain/frame-weight-template.hbs